File: go/test/fixtures/fixtures.go
package fixtures

import (
	. "shapes/base/models"
)

func SetupCircle() Circle {
	radius := 15.5
	return *Circle{}.New(radius)
}

func SetupRectangle() Rectangle {
	length := 22.75
	width := 33.25
	return *Rectangle{}.New(length, width)
}

func SetupSquare() Square {
	side := 10.25
	return *Square{}.New(side)
}

File: dart/test/fixtures/fixtures.dart
import "package:shapes/base/models.dart";

Circle setupCircle() {
  double radius = 15.5;
  return Circle(radius);
}

Rectangle setupRectangle() {
  double width = 22.75;
  double height = 33.25;
  return Rectangle(width, height);
}

Square setupSquare() {
  double side = 10.25;
  return Square(side);
}
